tools/bootstrap/README.md

```diff
@@ -0,0 +1,46 @@
+# Node.js Bootstrapping Guide
+
+## Windows
+
+A [Boxstarter][] script can be used for easy setup of Windows systems with all
+the required prerequisites for Node.js development. This script will install
+the following [Chocolatey] packages:
+  * [Git for Windows][] with the `git` and Unix tools added to the `PATH`
+  * [Python 2.x][]
+  * [Visual Studio 2017 Build Tools][] with [Visual C++ workload][]
+
+To install Node.js prerequisites using [Boxstarter WebLauncher][], just open
+[this link](http://boxstarter.org/package/nr/url?https://raw.githubusercontent.com/nodejs/node/master/tools/bootstrap/windows_boxstarter)
+with Internet Explorer or Edge browser on the target machine.
+
+Alternatively, you can use PowerShell. Run those commands from an elevated
+PowerShell terminal:
+```console
+Set-ExecutionPolicy Unrestricted -Force
+iex ((New-Object System.Net.WebClient).DownloadString('http://boxstarter.org/bootstrapper.ps1'))
+get-boxstarter -Force
+Install-BoxstarterPackage https://raw.githubusercontent.com/nodejs/node/master/tools/bootstrap/windows_boxstarter -DisableReboots
+```
+
+## Linux
+
+For building Node.js on Linux, following packets are required (note, that this
+can vary from distribution to distribution):
+  * `git-core`
+  * `python`
+  * `gcc-c++` or `g++`
+  * `make`
+
+To bootstrap Node.js on Linux, run in terminal:
+  * OpenSUSE: `sudo zypper install git-core python gcc-c++ make`
+  * Fedora: `sudo dnf install git-core python gcc-c++ make`
+  * Ubuntu, Debian: `sudo apt-get install git-core python g++ make`
```

Comment:
Does git-core work? From [the website](https://git-scm.com/download/linux) it seems like it should be `git` for all three distros.

Reply:
It looks like on openSUSE [git-core](https://software.opensuse.org/package/git-core) is more lightweight than [git](https://software.opensuse.org/package/git) package. But I think I'll change this anyway.